StorageAdapter/ValuesTest.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using Helpers;
+using Helpers.Http;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace StorageAdapter
+{
+    [Collection("Storage Adapter Value Tests")]
+    public class ValuesTest
+    {
+        private readonly IHttpClient httpClient;
+
+        public ValuesTest()
```

Comment:
I see that we are adding kv to the storage, but i didn't see any cleanup of data. I think we should add cleanup of added data so that the tests leave the storage clean for next run of the tests.

Reply:
Cleanup of data is added in the same pattern used by other tests implemented with IDisposable